tests/imgtype.go

```diff
@@ -79,60 +97,74 @@ func main() {
 
 		ref, err := is.Transport.ParseStoreReference(store, image)
 		if err != nil {
-			logrus.Fatalf("error parsing reference %q: %v", image, err)
-		}
-
-		src, err := ref.NewImageSource(systemContext, []string{expectedManifestType})
```

Comment:
Ah, yes.  Realized that we can grab the manifest contents from the Image object, so there's no need to create an ImageSource object to read it when we already have an Image object.